src/patch_computer.py

```
@@ -21,20 +23,36 @@ def to_array(final, ids):
+  for p in batch_pairs:
     surface_distance = np.linalg.norm(coords[:,p[0],:] - coords[:,p[1],:], axis=1) -(radius[p[0]] + radius[p[1]])
-    mean_dist.append(np.mean(surface_distance) if np.mean(surface_distance) >= 0 else 0)
+    batch_mean_dist.append(max(np.mean(surface_distance), 0))
+  return batch_mean_dist
+
+def initialize_worker(coords_, radius_):
+    global coords, radius
+    coords = coords_
+    radius = radius_
+
+def calc_distance_matrix(args, coords, radius, cores=min(max(os.cpu_count() - 1, 1), 16)):
```

Ruff (0.15.9)

[warning] Do not perform function call `min` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)